test/WebJobs.Script.Scaling.Tests/ScaleUtilsTests.cs

```diff
@@ -115,20 +116,28 @@ public void WorkersToDisplayStringTests()
         [Theory, MemberData("GetAndValidateTokenData")]
         public void GetAndValidateTokenTests(DateTime expiredUtc, bool expected)
         {
-            var token = ScaleUtils.GetToken(expiredUtc);
-
-            if (expected)
+            AppServiceSettings.RuntimeEncryptionKey = GenerateEncryptionKey();
```

Comment:
nit: We have a `TestScopedEnvironmentVariable` class that will allow you to exercise the logic to read the key from the environment variable.

Reply:
Great, that is handy.   However, our tests are in different assemblies and I prefer to keep the assembly independent nor referenced.   We have setter to allow mocking precisely that.